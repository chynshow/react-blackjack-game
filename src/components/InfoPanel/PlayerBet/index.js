import React from 'react';

const PlayerBet = ({ bet, className }) => {
  return <div className={className}>Player bet: {bet}$</div>;
};

export default React.memo(PlayerBet);
