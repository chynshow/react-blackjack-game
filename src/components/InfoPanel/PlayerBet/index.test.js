import React from 'react';
import PlayerBet from './index';
import { shallow } from 'enzyme';

it('render PlayerBet component', () => {
  const c = shallow(<PlayerBet />);
  expect(c).toMatchSnapshot();
});
