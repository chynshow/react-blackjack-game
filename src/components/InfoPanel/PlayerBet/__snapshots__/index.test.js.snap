// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PlayerBet component 1`] = `
<div>
  Player bet: 
  $
</div>
`;
